lang/rs/README.md

```diff
@@ -47,7 +47,7 @@ Following are the steps required to build Rust bindings, high level APIs and exa
       If no command line options are passed, the script will be executed with default values used in the script.
    4. Run loopback example:  `cd apis/cne`. Execute [./run_loopback.sh](./run_loopback.sh)
    5. Run echo server example: `cd examples/echo_server`. Execute [./run_echo_server.sh](./run_echo_server.sh)
-   6. Run packet forward example: `cd examples/fwd`. Execute [./run_fwd.h](./run_fwd.h) `<drop|fwd|tx-only|lb>`
+   6. Run packet forward example: `cd examples/fwd`. Execute [./run_fwd.h](./run_fwd.h) `<JSONC config file>` `<drop|fwd|tx-only|lb>`
```

Comment:
What about '[[drop | rx-only], tx-only, [fwd | forward], [lb | loopback]] we have shorter modes like fwd and lb which needs to be stated?

Reply:
I'll update the ReadME